lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1355,6 +1355,44 @@ def compute_timestamp(resource_type, record, time)
         ts_secs = timestamp.tv_sec
         ts_nanos = timestamp.tv_nsec
       end
+      ts_secs = begin
+                  Integer ts_secs
+                rescue ArgumentError, TypeError
+                  ts_secs
+                end
+      ts_nanos = begin
+                   Integer ts_nanos
+                 rescue ArgumentError, TypeError
+                   ts_nanos
+                 end
+
+      # Adjust timestamps from the future.
+      # There are two cases:
+      # 1. The parsed timestamp later in the current year:
```

Comment:
My fault. Should have been `timestamp is later`.

Reply:
Changed.